docs/articles/sde/how-to-cross-compile-using-qtcreator-in-sde.rst

```diff
@@ -0,0 +1,25 @@
+:orphan:
+
+How to cross compile using Qt Creator and PELUX SDE
+===================================================
+.. taglist:: howto
+
+Prerequisites:
+--------------
+#. Hardware target is running PELUX
+#. Hardware target has an sftp server installed
+
+Steps
+-----
+#. Open an instance of the PELUX SDE
```

Comment:
How do I do that? Do I doubleclick on something?

Reply:
This should be clear through the introduction